net.blix.BlueMail.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>net.blix.BlueMail</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>BlueMail</name>
+  <developer_name>Blix Inc.</developer_name>
+  <url type="homepage">https://bluemail.me</url>
+  <summary>BlueMail Email and Calendar App</summary>
+  <description>
+    <p>BlueMail by Blix is a free, beautifully designed, universal email app, capable of managing an unlimited number of mail accounts from various providers, allowing for smart push notifications and group emailing while enabling personalization across multiple email accounts. The BlueMail app connects directly to your mail server and is the perfect replacement for your stock email app.</p>
+    <p>NOTE: This package is not verified by, affiliated with, or supported by Blix Inc.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://www.bluemail.me/img/desktop/Bluemail-Linux-Application.png</image>
+      <caption>BlueMail for Linux</caption>
+    </screenshot>
+  </screenshots>
+  <launchable type="desktop-id">net.blix.bluemail.desktop</launchable>
```

Comment:
this is case sensitive, therefor it is incorrect, and appstream-compose wil fail causing the build error.